cmd/nodecmd/create.go

```diff
@@ -286,6 +292,48 @@ func setupBuildEnv(inventoryPath, ansibleHostIDs string) error {
 	return ansible.RunAnsiblePlaybookSetupBuildEnv(app.GetAnsibleDir(), inventoryPath, ansibleTargetHosts)
 }
 
+func GenerateStakingCertAndKey(nodesDirPath string) error {
+	certBytes, keyBytes, err := utils.NewStakingCertAndKeyBytes()
+	if err != nil {
+		return err
+	}
+	if err := os.WriteFile(filepath.Join(nodesDirPath, constants.StakerCertFileName), certBytes, 0o600); err != nil {
```

Comment:
use a new constant for 0o600 we have another one for 0o755 I believe follow that naming